### PR TITLE
GEOMESA-2636 Upgrade metrics to 3.2.6

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: build/cqs.tsv [hunk 1]
```diff
@@ -73,9 +73,9 @@ de.topobyte:osm4j-utils	0.0.22	compile
 de.topobyte:osm4j-xml	0.0.3	compile
 eu.medsea.mimeutil:mime-util	2.1.3	compile
 io.airlift:aircompressor	0.10	compile
-io.dropwizard.metrics:metrics-core	3.1.2	compile
-io.dropwizard.metrics:metrics-ganglia	3.1.2	compile
-io.dropwizard.metrics:metrics-graphite	3.1.2	compile
+io.dropwizard.metrics:metrics-core	3.2.6	compile
+io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
+io.dropwizard.metrics:metrics-graphite	3.2.6	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20220

Comment (elahrvivaz) [reply build/cqs.tsv]: @jnh5y could you ping the pmc for this one? Unless you want to wait and see how/if spring shakes out.

Comment (jnh5y) [reply build/cqs.tsv]: @elahrvivaz can we get a PR up for Spring first so that I can ask for all the CQs as once?

Comment (elahrvivaz) [reply build/cqs.tsv]: @jnh5y I opened a PR for spring here https://github.com/locationtech/geomesa/pull/2300, but it only requires piggy-back cqs so they should be automatically approved.

**Pull request merged**